doc/classes/AnimationNodeOneShot.xml

```diff
@@ -21,22 +43,30 @@
 			If [member autorestart] is [code]true[/code], a random additional delay (in seconds) between 0 and this value will be added to [member autorestart_delay].
 		</member>
 		<member name="fadein_time" type="float" setter="set_fadein_time" getter="get_fadein_time" default="0.0">
+			The fade-in duration. For example, setting this to [code]1s[/code] for a 5 second length animation will produce a crossfade that starts at 0 second and ends at 1 second during the animation.
```

Comment:
Code markup should match what you actually write in code, which wouldn't be `1s` here but `1.0`.